<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JointsController</class>
 <widget class="QWidget" name="JointsController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>617</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0" colspan="5">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="5">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="5">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="5">
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="5">
     <widget class="Line" name="line_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="15" column="0" colspan="5">
     <widget class="Line" name="line_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="17" column="0" colspan="5">
     <widget class="Line" name="line_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="19" column="0" colspan="5">
     <widget class="Line" name="line_8">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="21" column="0" colspan="5">
     <widget class="Line" name="line_9">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="23" column="0" colspan="5">
     <widget class="Line" name="line_10">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="5">
     <widget class="Line" name="line_11">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="13" column="0" colspan="5">
     <widget class="Line" name="line_12">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="25" column="0" colspan="5">
     <widget class="Line" name="line_13">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="26" column="2">
     <widget class="QCheckBox" name="checkStiffness">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>Enable or Disable stiffness,
checkbox is not sync'd with the robot stiffness. Capture will update it for the value of that moment.
If one or more joint stiffness value is not egual to the one of all the others an &quot;unset&quot; (neither checked or unchecked) is shown and the aside value loose meaning. set again to uniform all again.</string>
      </property>
      <property name="text">
       <string>Stiffness</string>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="26" column="3">
     <widget class="QDoubleSpinBox" name="spinStiffness">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="12" column="4">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>30</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="4">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>30</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="26" column="0" colspan="2">
     <widget class="QPushButton" name="btnUpdateJoints">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Capture</string>
      </property>
     </widget>
    </item>
    <item row="27" column="3">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_23">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>LEFT</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_24">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>RIGHT</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string>Shouler Pitch</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="label_26">
      <property name="text">
       <string>Shoulder Roll</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QLabel" name="label_27">
      <property name="text">
       <string>Elbow Yaw</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <widget class="QLabel" name="label_28">
      <property name="text">
       <string>Elbow Roll</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="14" column="2">
     <widget class="QLabel" name="label_29">
      <property name="text">
       <string>Hip Yaw Pitch</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="16" column="2">
     <widget class="QLabel" name="label_30">
      <property name="text">
       <string>Hip Roll</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="18" column="2">
     <widget class="QLabel" name="label_31">
      <property name="text">
       <string>Hip Pitch</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="20" column="2">
     <widget class="QLabel" name="label_32">
      <property name="text">
       <string>Knee Pitch</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="22" column="2">
     <widget class="QLabel" name="label_33">
      <property name="text">
       <string>Ankle Pitch</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="24" column="2">
     <widget class="QLabel" name="label_34">
      <property name="text">
       <string>Ankle Roll</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="DoublePicker" name="pickerLShoulderPitch" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="DoublePicker" name="pickerLShoulderRoll" native="true"/>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="DoublePicker" name="pickerLElbowYaw" native="true"/>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="DoublePicker" name="pickerLElbowRoll" native="true"/>
    </item>
    <item row="14" column="0" colspan="2">
     <widget class="DoublePicker" name="pickerLHipYawPitch" native="true"/>
    </item>
    <item row="16" column="0" colspan="2">
     <widget class="DoublePicker" name="pickerLHipRoll" native="true"/>
    </item>
    <item row="18" column="0" colspan="2">
     <widget class="DoublePicker" name="pickerLHipPitch" native="true"/>
    </item>
    <item row="20" column="0" colspan="2">
     <widget class="DoublePicker" name="pickerLKneePitch" native="true"/>
    </item>
    <item row="22" column="0" colspan="2">
     <widget class="DoublePicker" name="pickerLAnklePitch" native="true"/>
    </item>
    <item row="24" column="0" colspan="2">
     <widget class="DoublePicker" name="pickerLAnkleRoll" native="true"/>
    </item>
    <item row="24" column="3" colspan="2">
     <widget class="DoublePicker" name="pickerRAnkleRoll" native="true"/>
    </item>
    <item row="22" column="3" colspan="2">
     <widget class="DoublePicker" name="pickerRAnklePitch" native="true"/>
    </item>
    <item row="20" column="3" colspan="2">
     <widget class="DoublePicker" name="pickerRKneePitch" native="true"/>
    </item>
    <item row="18" column="3" colspan="2">
     <widget class="DoublePicker" name="pickerRHipPitch" native="true"/>
    </item>
    <item row="16" column="3" colspan="2">
     <widget class="DoublePicker" name="pickerRHipRoll" native="true"/>
    </item>
    <item row="14" column="3" colspan="2">
     <widget class="DoublePicker" name="pickerRHipYawPitch" native="true"/>
    </item>
    <item row="10" column="3" colspan="2">
     <widget class="DoublePicker" name="pickerRElbowRoll" native="true"/>
    </item>
    <item row="8" column="3" colspan="2">
     <widget class="DoublePicker" name="pickerRElbowYaw" native="true"/>
    </item>
    <item row="6" column="3" colspan="2">
     <widget class="DoublePicker" name="pickerRShoulderRoll" native="true"/>
    </item>
    <item row="4" column="3" colspan="2">
     <widget class="DoublePicker" name="pickerRShoulderPitch" native="true"/>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="DoublePicker" name="pickerHeadYaw" native="true"/>
    </item>
    <item row="0" column="3" colspan="2">
     <widget class="DoublePicker" name="pickerHeadPitch" native="true"/>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>27</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Bitstream Charter</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Bitstream Charter'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans';&quot;&gt;Head&lt;br /&gt;Yaw     -    Pitch&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="12" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>100</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DoublePicker</class>
   <extends>QWidget</extends>
   <header location="global">DoublePicker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
